holoviews/plotting/mpl/renderer.py

```diff
@@ -318,3 +321,4 @@ def load_nb(cls, inline=True):
         """
```

Comment:
As the bokeh renderer also has a ``load_nb`` classmethod (and notebooks are explicitly mentioned in that method name), I would consider making ``_notebook`` into a constant ``notebook_context`` class parameter. The docstring can say it is set to True if the ``load_nb`` classmethod has been called.